### ISSUE TITLE
Minimum Memory Requirement of GPU in Training Model

### ISSUE THREAD

**Issue by HailinRen** [michalfaber/keras_Realtime_Multi-Person_Pose_Estimation#52]
Dear All,

I am wondering what's the minimum gpu memory requirement for training the model. I have tried on gtx 970 and run out of all 4GB memory. Could anyone tell me the minimum memory requirement for successful training?

Thank you!

**Comment by anatolix** [comment]
Lower batch size (by default it is 10) and you most probably fit in memory. 
My gtx1080ti with 11Gb allows maximum batch=20, so probably requirements are 
0.5Gb per item + some const.

NB: you should lower learning rate accordingly i.e. batch size=5, lower it two times.
Most probably this will slower you training speed too.
